backend/tracim_backend/command/cleanup.py

```diff
@@ -166,36 +168,44 @@ def take_app_action(self, parsed_args: argparse.Namespace, app_context: AppEnvir
             )
             print("~~~~")
             print("Deletion of Caldav Agenda\n")
-            # INFO - G.M - 2019-12-13 - cleanup agenda at end of process
-            if deleted_workspace_ids:
-                deleted_workspace_ids_str = [
-                    '"{}"'.format(workspace_id) for workspace_id in deleted_workspace_ids
-                ]
-                print("delete agenda of workspaces {}".format(", ".join(deleted_workspace_ids_str)))
-                for workspace_id in deleted_workspace_ids:
-                    try:
-                        cleanup_lib.delete_workspace_agenda(workspace_id)
-                    except AgendaNotFoundError:
-                        print(
-                            'Warning: Cannot delete agenda for workspace "{}", agenda not found. Agenda path may be incorrect or agenda not created'.format(
-                                workspace_id
-                            )
+            app_lib = ApplicationApi(app_list=app_list)
+            if app_lib.exist("agenda"):
```

Comment:
Must-use constants for "core" Tracim applications.

Reply:
done.